Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -16696,6 +16697,24 @@ void WebPageProxy::documentURLForConsoleLog(WebCore::FrameIdentifier frameID, Co
     completionHandler({ });
 }
 
+void WebPageProxy::reportMixedContentViolation(FrameIdentifier frameID, bool blocked, const URL& target, bool isUpgradingIPAddressAndLocalhostEnabled)
```

Comment:
It looks like `isUpgradingIPAddressAndLocalhostEnabled` is set from `document->settings().iPAddressAndLocalhostMixedContentUpgradeTestingEnabled()`, so we might use `protectedPreferences()->iPAddressAndLocalhostMixedContentUpgradeTestingEnabled()` and remove the boolean from message (UI process should rely on its own record if possible)